test/models/mobilenetv2_nchw/mobilenetv2.js

```diff
@@ -0,0 +1,164 @@
+'use strict';
+import * as utils from '../../utils.js';
+
+const url = import.meta.url;
+const assert = chai.assert;
+
+describe('test mobilenetv2 nchw', function() {
+  // eslint-disable-next-line no-invalid-this
+  this.timeout(0);
+  let graph;
+  let beforeNumBytes;
+  let beforeNumTensors;
+  before(async () => {
+    if (typeof _tfengine !== 'undefined') {
+      beforeNumBytes = _tfengine.memory().numBytes;
+      beforeNumTensors = _tfengine.memory().numTensors;
+    }
+    const context = navigator.ml.createContext();
+    const builder = new MLGraphBuilder(context);
+
+    async function buildConv(input, name, clip = true, options = undefined) {
+      const prefix = './weights/conv_' + name;
+      const weightsName = prefix + '_weight.npy';
+      const weights =
+          await utils.buildConstantFromNpy(builder, new URL(weightsName, url));
+      const biasName = prefix + '_bias.npy';
+      const bias =
+          await utils.buildConstantFromNpy(builder, new URL(biasName, url));
+      const conv = builder.add(
+          builder.conv2d(input, weights, options),
+          builder.reshape(bias, [1, -1, 1, 1]));
+      if (clip) {
+        return builder.clamp(
+          conv,
+          {
+            minValue: builder.constant(0.),
+            maxValue: builder.constant(6.0),
+          },
+        );
+      } else {
+        return conv;
+      }
+    }
+
+    async function buildGemm(input, name) {
+      const prefix = './weights/gemm_' + name;
+      const weightsName = prefix + '_weight.npy';
+      const weights =
+          await utils.buildConstantFromNpy(builder, new URL(weightsName, url));
+      const biasName = prefix + '_bias.npy';
+      const bias =
+          await utils.buildConstantFromNpy(builder, new URL(biasName, url));
+      const options = {c: bias, bTranspose: true};
+      return builder.gemm(input, weights, options);
+    }
+
+    async function buildFire(
```

Comment:
Please align to the implementation of mobilenet example, as my comment there https://github.com/webmachinelearning/webnn-samples/pull/37#discussion_r614522995

Reply:
@huningxin Thanks your suggestion.
@Honry Since you have updated your code of https://github.com/webmachinelearning/webnn-samples/pull/37, would you please also help update code here, thanks.

Reply:
@BruceDai, no problem, I will update this PR once the https://github.com/webmachinelearning/webnn-samples/pull/37 is landed.